README.md

```diff
@@ -1,50 +1,37 @@
-DNS query functions
-===================
+## Table of Contents
 
-This module contains query functions for DNS for use from Puppet.
+1. [Overview](#overview)
```

Comment:
Given the size of the readme, isn't it better to skip the table of contents? GitHub has also introduced it natively: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/ so the practice may be a bit outdated.

Reply:
done